download_and_create_reference_datasets/v02/create_ht__cadd.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
```

Comment:
This is inconsistent with the other v02 scripts, which use python3.

Reply:
oops

Reply:
fixed in  https://github.com/macarthur-lab/hail-elasticsearch-pipelines/commit/08e22d61390457c02929d8372cf80eebf041bc3d